packages/dm-core-plugins/src/grid/GridPlugin.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  Loading,
+  TGenericObject,
+  useDocument,
+} from '@development-framework/dm-core'
+
+import React from 'react'
+import styled from 'styled-components'
+import { TGridSize, TGridPluginConfig } from './types'
+import { GridItems } from './GridItems'
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(${(props: TGridSize) => props.columns}, 1fr);
+  grid-template-rows: repeat(${(props: TGridSize) => props.rows}, 1fr);
+  grid-column-gap: 5px;
+  grid-row-gap: 5px;
+`
+
+export const GridPlugin = (props: TGridPluginConfig): JSX.Element => {
```

Comment:
This is a different method then how it's done in TabsPlugin, where `props: IUIPlugin` and config is set to default values in the beginning of the function. Should we standardize this?